etc/sync-function-template.js

```diff
@@ -796,14 +796,34 @@ function synctos(doc, oldDoc) {
 
   var theDocDefinition = docDefinitions[theDocType];
 
+  if (theDocDefinition.customActions && typeof(theDocDefinition.customActions.onTypeIdentificationSucceeded) === 'function') {
+    theDocDefinition.customActions.onTypeIdentificationSucceeded(doc, oldDoc);
```

Comment:
I elected to pass the `oldDoc` parameter completely unmodified in all cases instead of replacing it with `null` if its `_deleted` property is `true` to allow for the greatest level of flexibility for custom actions. The README makes note of this so it's not a surprise to a user of synctos.